.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature request
+description:  |
+  💡 Suggest an idea for the `NextUI` project
+  Examples
+    - propose a new component
+    - improve an exiting component
+    - ....etc
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue form is for requesting features only! For example, requesting a new component, behavior ... etc
+        If you want to report a bug, please use the [bug report](https://github.com/nextui-org/nextui/issues/new?assignees=&labels=&template=bug_report.yml) form.
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description: A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe the solution you'd like
+      description: A clear and concise description of what you want to happen.
+      placeholder: |
+        As a user, I expected ___ behavior but ___ ...
+        
+        Ideal Steps I would like to see:
+        1. Go to '...'
+        2. Click on '....'
+        3. ....
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe alternatives you've considered
+      description: A clear and concise description of any alternative solutions or features you've considered.
+  - type: textarea
+    attributes:
+      label: Screenshots or Videos
+      description: |
+        If applicable, add screenshots or a video to help explain your problem.
+        For more information on the supported file image/file types and the file size limits, please refer
+        to the following link: https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files
```

Comment:
**_Same as in Bug Report_**

![image](https://user-images.githubusercontent.com/47073516/156115698-48d151db-b2f5-4a1d-8b73-81267fc2298f.png)

Reply:
I'm a bit reluctant to make this a hyper link (not opposed) I think one benefit of having  the long URL is that i'ts harder for people to miss - and the goal is to reduce friction for maintainers and make resources easily accessible (hard to miss) for contributors